src/status_im2/contexts/quo_preview/inputs/title_input.cljs

```diff
@@ -0,0 +1,54 @@
+(ns status-im2.contexts.quo-preview.inputs.title-input
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [quo2.core :as quo]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label "disabled?"
+    :key   :disabled?
+    :type  :boolean}
+   {:label "blur?"
+    :key   :blur?
+    :type  :boolean}
+  ])
+
+(defn cool-preview
+  []
+  (let [state          (reagent/atom {:blur?     false
+                                      :disabled? false})
+        value          (reagent/atom "")
+        max-length     24
+        on-change-text (fn [v]
+                         (reset! value v)
+                         (reagent/flush)
+                       )]
+    (fn []
+      [rn/touchable-without-feedback {:on-press rn/dismiss-keyboard!}
+       [rn/view {:padding-bottom 150}
+        [rn/view {:flex 1}]
+        [preview/customizer state descriptor]
+        [rn/view
+         {:padding-vertical  60
+          :flex-direction    :row
+          :margin-horizontal 20
+          :justify-content   :center}
+         [quo/title-input
+          {:max-length     max-length
+           :on-change-text on-change-text
+           :disabled?      (:disabled? @state)
+           :blur?          (:blur? @state)
+           :placeholder    "type something here"} @value]]]])))
```

Comment:
Related to code style, I'd put that `@value` in the following line (it'd be 43)  instead of right after the map.

Reply:
This is in the Clojure Style Guide, so we should fix that https://guide.clojure.style/#vertically-align-fn-args.

I've seen this stylistic issue in lots of PRs and I always paste the style guide link to help spread the word. It's a shame we can't automate with zprint. Or can we? That tool has so many options that perhaps it can actually do that for us...